build/dnscrypt-proxy.toml

```diff
@@ -235,7 +235,7 @@ block_ipv6 = false
 ## example.com 9.9.9.9
 ## example.net 9.9.9.9,8.8.8.8,1.1.1.1
 
-# forwarding_rules = 'forwarding-rules.txt'
+forwarding_rules = 'forwarding-rules.txt'
```

Comment:
Could you add a comment above explaining the reasoning of the contents of `forwarding-rules.txt`? Same for `cloaking-rules.txt`

Reply:
Forwarding rules make sure that .dappnode and .eth domains get resolved by Docker's internal DNS. Cloaking rules make sure that we override resolution of xxxxxx.dydns.dappnode.io from public to private IP.